pulsar-broker/src/main/java/org/apache/pulsar/client/impl/RawReaderImpl.java

```diff
@@ -52,12 +54,12 @@
     private final ConsumerConfiguration consumerConfiguration;
     private RawConsumerImpl consumer;
 
-    public RawReaderImpl(PulsarClientImpl client, String topic, CompletableFuture<Consumer> consumerFuture) {
+    public RawReaderImpl(PulsarClientImpl client, String subscription,
```

Comment:
topic, subscription is a better sequence than subscription, topic.

Reply:
will change.